test/Integration/test-csx/Common/utils.csx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Is this file materially different from utils.cs elsewhere in test code? Can it be reused?

Reply:
that's testutils and the functions in use aren't there, so added a new one here - but I finally figured out on how to use the utils from extension itself.